packages/11ty/_plugins/figures/index.js

```diff
@@ -0,0 +1,17 @@
+const Figure = require('./figure')
+const iiifConfig = require('./config')
+
+module.exports = function (eleventyConfig, options = {}) {
+  eleventyConfig.addGlobalData('iiifConfig', iiifConfig(eleventyConfig))
```

Comment:
Since the `config` module exports a function that expects the `eleventyConfig` as an argument updating the global data with with a `iiif` or `iiifConfig` property may be a concern of the `config` module.
```suggestion
  config(eleventyConfig)
```